src/vos/tests/evt_ctl.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-if [ "$USE_VALGRIND" = "yes" ]; then
+if [ "$USE_VALGRIND" = "memcheck" ]; then
+    VCMD="valgrind --leak-check=full --show-reachable=yes --error-limit=no \
+          --gen-suppressions=all --suppressions=${VALGRIND_SUPP}"
+elif [ "$USE_VALGRIND" = "pmemcheck" ]; then
```

Comment:
With this change are we doing valgrind testing on each PR run (both memcheck and pmemcheck)?

Reply:
No, this patch is only about memcheck. I'll send another one for pmemcheck later.